src/platforms/php/common/performance.mdx

```diff
@@ -112,11 +112,23 @@ Currently, we recommend using Relay in `managed` mode (which is the default). Th
 Don't forget to update your `DSN` to point to your running Relay instance.
 After you set up Relay, you should see a dramatic improvement to the impact on your server.
 
+<PlatformSection notSupported={["php.laravel"]}>
+
+## Connecting Services
+
+If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/) depending on where your request originates there are two methods to connect your traces:
+1. For requests that start in your backend, this is done by [adding a meta tag](/platforms/javascript/performance/connect-services/#pageload) in your HTML template that contains tracing information.
+2. For requests that start in JavaScript, this is done by [setting a header](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests) on requests to your backend.
+
+Otherwise backend services with Performance Monitoring will connect automatically.
+
+</PlatformSection>
+
 <PlatformSection supported={["php.laravel"]}>
 
-### Connect Your Frontend
+## Connecting Services
```

Comment:
Took me some time to realize we duplicate the header but within `<PlatformSection />`.
Since the header should always be the same, could we move it above the first `<PlatformSection />`?

---

```md
## Connecting Services

<PlatformSection notSupported={["php.laravel"]}> ... </>

<PlatformSection supported={["php.laravel"]}> ... </>
```